tests/org.eclipse.swt.tests/ManualTests/org/eclipse/swt/tests/manual/ColorProviderTest.java

```diff
@@ -0,0 +1,116 @@
+package org.eclipse.swt.tests.manual;
+
+import org.eclipse.swt.*;
+import org.eclipse.swt.graphics.Point;
+import org.eclipse.swt.layout.*;
+import org.eclipse.swt.widgets.*;
+
+public class ColorProviderTest {
```

Comment:
NIT: since this isn't an automatic test class and since we have this kind of showcases already (`ControlExample` / `CustomControlExample`), how about calling it:

```suggestion
public class ColorProviderTExample {
```
?

Also, would it be possible to add the ´darkChkBx´ to the `ControlExample` and `CustomControlExample`? Those 2 classes already showcase all of the controls so it should keep up-to-date while we keep implementing the widgets

Reply:
IMHO we are not yet at this point. Maybe if Composite also is a custom control.

Reply:
I've renamed it to ColorProviderSnippet.